DeUrgenta/Features/EmergencyBackpack/View/MyBackpackView.swift

```diff
@@ -0,0 +1,79 @@
+import SwiftUI
+
+struct MyBackpackView: View {
+    let items = [
+        CategoryItem(title: AppStrings.CategoryItem.foodTitle.localized(), icon: "icon-syrup", type: .food),
+        CategoryItem(title: AppStrings.CategoryItem.hygieneItemsTitle.localized(), icon: "icon-cotton-swabs", type: .hygieneItems),
+        CategoryItem(title: AppStrings.CategoryItem.firstAidKitTitle.localized(), icon: "icon-capsules", type: .firstAidKit),
+        CategoryItem(title: AppStrings.CategoryItem.docsTitle.localized(), icon: "icon-prescription", type: .docs),
+        CategoryItem(title: AppStrings.CategoryItem.survivalItemsTitle.localized(), icon: "icon-medical-mask", type: .survivalItems),
+        CategoryItem(title: AppStrings.CategoryItem.mixedTitle.localized(), icon: "icon-pharmacy", type: .mixed),
+    ]
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                Text(AppStrings.MyBackpackView.headerTitle.localized())
+                    .font(.custom("IBMPlexSans-SemiBold", size: 23))
+                    .frame(alignment: .leading)
+                    .padding(.vertical, 10)
+
+                VStack {
+                    Text(AppStrings.MyBackpackView.firstParagraph.localized())
+                        .font(.custom("IBMPlexSans-Regular", size: 16))
```

Comment:
.font(.custom("IBMPlexSans-Regular", size: 16))
.foregroundColor(.darkText)

Seems to repeat - can we define a custom modifier to have a default Text style or something similar?